buildSrc/src/main/java/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -109,7 +109,7 @@ public void apply(Project project) {
                     .setFreeCompilerArgs(
                         ImmutableList.of("-module-name", kotlinModuleName(project))));
 
-    Dokka.configure(project, android, firebaseLibrary);
```

Comment:
Is `Dokka.java` still needed or can it be removed?

Reply:
It can be removed, should I go ahead and remove it?

Reply:
I think we should, but a follow up PR might be better, since this one is already large

Reply:
Agreed 
b/243685887